### ISSUE TITLE
Some questions If I plug it into yolov8 and use it,pls

### ISSUE THREAD

**Issue by crisych** [NUS-HPC-AI-Lab/InfoBatch#24]
Hi,if i use it in yolov8,What would happen?
I mean, will the  parameters decrease or will just increase the speed of training and shorten the reasoning time?
Also, could you show me how to use it in yolov8,thanks.

**Comment by henryqin1997** [comment]
Hi, thank you for the interest. We are currently working on the public example. Previously our contributor has verified it on private data; we will release the public example this week or next.

**Comment by crisych** [comment]
Thanks for your reply, I will stay tuned

**Comment by crisych** [comment]
Hi, take the liberty of asking, when will the example about yolov8 be released?I can't wait to study

**Comment by henryqin1997** [comment]
Sorry, just got to be suddenly busy and miss the time. I will continue on it next week.

**Comment by henryqin1997** [comment]
@Zhou2019 @crisych @tiandunx  I have adapted the loss and training run for ultralytics YOLO. See https://github.com/henryqin1997/ultralytics-infobatch . Do you have hyperparameters so that I can test whether it reproduce lossless result with reported saving? I will update the document this weekend to show how I adapt it. Right now one can find it in study_case.ipynb and commit records.
